src/platform/w3dengine/client/w3ddisplay.cpp

```diff
@@ -16,27 +16,70 @@
 #include "w3ddisplay.h"
 #include "assetmgr.h"
 #include "baseheightmap.h"
+#include "colorspace.h"
 #include "displaystring.h"
 #include "displaystringmanager.h"
+#include "drawable.h"
+#include "drawmodule.h"
 #include "dx8wrapper.h"
 #include "filesystem.h"
+#include "gameclient.h"
+#include "gameengine.h"
+#include "gamelogic.h"
+#include "gametext.h"
 #include "globaldata.h"
+#include "globallanguage.h"
+#include "hlod.h"
 #include "image.h"
+#include "ingameui.h"
+#include "keyboard.h"
+#include "line2d.h"
+#include "main.h"
+#include "mesh.h"
+#include "meshmdl.h"
+#include "mouse.h"
+#include "network.h"
+#include "particlesysmanager.h"
+#include "physicsupdate.h"
 #include "predlod.h"
 #include "renderdevicedesc.h"
 #include "rtsutils.h"
+#include "scriptengine.h"
 #include "shadermanager.h"
+#include "sortingrenderer.h"
 #include "videobuffer.h"
-#include "view.h"
 #include "w3d.h"
+#include "w3ddebugdisplay.h"
+#include "w3ddynamiclight.h"
+#include "w3dfilesystem.h"
+#include "w3dprojectedshadow.h"
+#include "w3dscene.h"
+#include "w3dshroud.h"
+#include "w3dterraintracks.h"
+#include "w3dview.h"
+#include "w3dwater.h"
+#include "water.h"
+#include "worldheightmap.h"
+#ifdef BUILD_WITH_D3D8
+#include <io.h>
+#endif
+#include <stdio.h>
+
+#ifdef GAME_DEBUG_STRUCTS
+ICoord2D &s_leftClickReleased = (ICoord2D &)0x00E1B118;
```

Comment:
Do this through the normal `Make_Global` otherwise it will easily be missed later on.

Reply:
Fixed.